pkg/buildplan/mock_test.go

```diff
@@ -9,14 +12,15 @@ import "github.com/ActiveState/cli/pkg/buildplan/raw"
 //
 // The artifact cycle is:
 // 00000000-0000-0000-0000-000000000001
+//
 //	-> 00000000-0000-0000-0000-000000000002
 //	  -> 00000000-0000-0000-0000-000000000003
 //	    -> 00000000-0000-0000-0000-000000000001 (Cycle back to the first artifact)
 func createMockArtifactWithCycles() *Artifact {
 	// Create the artifacts with placeholders
-	artifact0001 := &Artifact{ArtifactID: "00000000-0000-0000-0000-000000000001"}
-	artifact0002 := &Artifact{ArtifactID: "00000000-0000-0000-0000-000000000002"}
-	artifact0003 := &Artifact{ArtifactID: "00000000-0000-0000-0000-000000000003"}
+	artifact0001 := &Artifact{ArtifactID: "00000000-0000-0000-0000-000000000001", MimeType: types.XArtifactMimeType}
```

Comment:
I added these mime types when I was trying to filter on them. Are they still needed?